src/widget.ts

```diff
@@ -72,7 +72,7 @@ export class SpreadsheetWidget extends Widget {
   }
 
   protected parseValue(content: string): jexcel.CellValue[][] {
-    const parsed = Papa.parse(content, { delimiter: this.separator });
+    const parsed = Papa.parse<string[]>(content, { delimiter: this.separator });
```

Comment:
This is needed because of https://github.com/DefinitelyTyped/DefinitelyTyped/issues/52064